src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -226,10 +228,16 @@ class ContentSidebar extends PureComponent<Props, State> {
      * @param {Error} error - Error object
      * @return {void}
      */
-    errorCallback = (error: $AxiosXHR<any>): void => {
+    errorCallback = (error: ElementsXhrError): void => {
         /* eslint-disable no-console */
         console.error(error);
         /* eslint-enable no-console */
+
+        /* eslint-disable react/prop-types */
+        this.props.onError(error, ERROR_CODE_FETCH_FILE, true, {
```

Comment:
If you destructure `onError` per @priyajeet's suggestion, does eslint still complain?

Reply:
Yes it does, weird.